README.md

```diff
@@ -8,20 +8,27 @@
 
 ## Introduction
 
-In JavaScript, as in other Object Oriented languages, we've learned
-that you can create `class`es and build methods that can perform
-actions on instance data, or specific to the `class`. What if you have
-`class`es that exhibit many of the same behaviors, such as `Cat`, `Dog`,
-and `Bird`, which all have a method for `speak`? In JavaScript, you can
-create "child" object `class`es that inherit methods from their "parent"
-`class`es. In this lesson, we'll discuss 2 ways of _extending_ functionality
-to other `class`es.
+(KG: i love that you're playing by the rules, but don't over-generalize: use ``
+for code-words `class` is what you type...versus "classes exist in OO
+languages")
+
+In JavaScript, as in other Object Oriented languages, we've learned that you
+can create classes and build methods that can perform actions on instance data,
+or specific to the class.
+
+What if you have several classes that exhibit many of the same behaviors, such
+as `Cat`, `Dog`, and `Bird`, all of which have a method for `speak`? In
+JavaScript, you can create "child" classes that inherit methods from their
+"parent" classes.
+
+In this lesson, we'll discuss 2 ways of _extending_ functionality from a
```

Comment:
@drakeltheryuujin Note how i'm using vertical whitespace to keep concepts small and chunked for n00bs.